das/daser.go

```diff
@@ -47,7 +46,7 @@ func NewDASer(
 		hsub:   hsub,
 		getter: getter,
 		cstore: wrappedDS,
-		jobsCh: make(chan *catchUpJob),
+		jobsCh: make(chan *catchUpJob, 4), // TODO @renaynay: is this a reasonable buffer?
```

Comment:
Let's set this to 16 so we have some legroom. If the buffer is full, the `sample` will stall until the catchUpManager finishes a job. The bigger the buffer, the lesser chances for `sample` to stall